### PR TITLE
847061-UG documentation for Treeview mvvm and RTL

### PR DESCRIPTION
**Pull request by Dhivya-SF4094**
Prepared UG documentation for treeview mvvm and RTL

### CODE REVIEW

File: MAUI/TreeView/mvvm.md [hunk 1]
```diff
@@ -0,0 +1,397 @@
...
+    {          
+        App.Current.MainPage.DisplayAlert("Alert", ((obj as TreeViewNode).Content as FileManager).ItemName+ "  is LongPressed","OK");
+    }   
+}
+{% endhighlight %}
+{% endtabs %}
+
+### Expand command
+
+The `ExpandCommand` will be triggered while expanding the node and passing the `TreeViewNode` as command parameter. TreeView expands the node based on the return value of `CanExecute` method implementation of `ExpandCommand`. If you return false, then expand action will be canceled. Execute method implementation of `ExpandCommand` will get called after expanding of node.
```
Comment (M-A32) [inline MAUI/TreeView/mvvm.md:204]: The `ExpandCommand` will be triggered while expanding the node and passing the `TreeViewNode` as a command parameter. TreeView expands the node based on the return value of the `CanExecute` method implementation of `ExpandCommand`. If you return false, then expand action will be canceled. Execute method implementation of `ExpandCommand` will get called after expanding the node.

File: MAUI/TreeView/right-to-left.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: Right To Left in .NET MAUI TreeView Control | Syncfusion
+description: Learn here all about Right To Left support in Syncfusion .NET MAUI TreeView (SfTreeView) Control and more.
```
Comment (M-A32) [inline MAUI/TreeView/right-to-left.md:4]: description: Learn here about Right To Left support in Syncfusion .NET MAUI TreeView (SfTreeView) Control and more.

**Review by M-A32 — CHANGES_REQUESTED** [review]
I have completed the review.

File: MAUI/TreeView/mvvm.md [hunk 3]
```diff
@@ -0,0 +1,397 @@
+---
+layout: post
+title: Working with MVVM in .NET MAUI TreeView Control | Syncfusion
+description: Learn here all about Working with MVVM support in Syncfusion .NET MAUI TreeView (SfTreeView) Control and more.
```
[the remark this answers is not in the document]
Comment (Dhivya-SF4094) [reply MAUI/TreeView/mvvm.md:4]: Replaced with the above content.

File: MAUI/TreeView/mvvm.md [hunk 4]
```diff
@@ -0,0 +1,397 @@
...
+        SelectedPlace = nsw;
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+### Binding SelectedItems
+
+TreeView support to select multiple items through binding the `SelectedItems` property from view model with `ObservableCollection<object>` type.
```
[the remark this answers is not in the document]
Comment (Dhivya-SF4094) [reply MAUI/TreeView/mvvm.md:76]: Replaced with the above content.

File: MAUI/TreeView/mvvm.md [hunk 5]
```diff
@@ -0,0 +1,397 @@
...
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## Commands
+
+### Tap command
+
+The `TapCommand` will be triggered whenever tapping the item and passing the `TreeViewNode` as command parameter.
```
[the remark this answers is not in the document]
Comment (Dhivya-SF4094) [reply MAUI/TreeView/mvvm.md:140]: Replaced with the above content in TapCommand

File: MAUI/TreeView/mvvm.md (hunk 1 above)
Comment (Dhivya-SF4094) [reply MAUI/TreeView/mvvm.md:204]: Replaced with the above content in Expand command

File: MAUI/TreeView/mvvm.md [hunk 6]
```diff
@@ -0,0 +1,397 @@
...
+    {
+        App.Current.MainPage.DisplayAlert("Alert", ((obj as TreeViewNode).Content as FileManager).ItemName+ "  is Expanded","OK");
+    }   
+}
+{% endhighlight %}
+{% endtabs %}
+
+### Collapse command
+
+The `CollapseCommand` will be triggered while collapsing the node and passing the `TreeViewNode` as command parameter. TreeView collapses the node based on the return value of `CanExecute` method implementation of `CollapseCommand`. If you return false, then collapse action will be canceled. `Execute` method implementation of `CollapseCommand` will be called after the node has collapsed.
```
[the remark this answers is not in the document]
Comment (Dhivya-SF4094) [reply MAUI/TreeView/mvvm.md:267]: Replaced with the above content in Collapse command

File: MAUI/TreeView/right-to-left.md (hunk 2 above)
Comment (Dhivya-SF4094) [reply MAUI/TreeView/right-to-left.md:4]: Changed the description with the above content.

**Pull request merged**